tests/microos/rebuild_initrd.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Rebuild initrd without ignition and combustion
```

Comment:
If you want, you could add also the reason of this module here as well, not only in the commit message. It would help anyone stepping into this module for the first time.

Reply:
Added